R/api-config.R

```diff
@@ -3,6 +3,7 @@
 #' @description  Global configuration for the OSPSuite .NET API
 ApiConfig <- R6::R6Class(
   "ApiConfig",
+  cloneable = FALSE,

```

Comment:
@PavelBal Good to know here. By default it is true and it is possible to clone an object which would most likely be a complete mess with the .NET references. I have disabled it for all object inheriting from .NET wrapper or referencing any .NET object